api/types/image.go

```diff
@@ -12,10 +12,15 @@ type Image struct {
 	// NewName is the value used to replace the original name.
 	NewName string `json:"newName,omitempty" yaml:"newName,omitempty"`
 
+	// TagSuffix is the value used to suffix the original tag
+	// If digest or newtag is present TagSuffix value is ignored
```

Comment:
This comment is no longer accurate, could you update it along with the one for Digest? Other than that this PR LGTM

Reply:
okay done